pkg/shares/non_interactive_defaults_test.go

```diff
@@ -0,0 +1,260 @@
+package shares
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/tendermint/tendermint/pkg/consts"
+)
+
+func TestMsgSharesUsedNIDefaults(t *testing.T) {
+	type test struct {
+		cursor, squareSize, expected int
+		msgLens                      []int
+		indexes                      []uint32
+	}
+	tests := []test{
+		{2, 4, 1, []int{1}, []uint32{2}},
```

Comment:
[Non Blocking][Question]
Awesome tests :tada: :rocket:. However, I guess they will need to change depending on the outcome of this https://github.com/celestiaorg/celestia-app/pull/673#discussion_r961265438

Reply:
why would this need to change? I'm not sure what is meant by strictly in that sentence

Reply:
If a message length is 2 shares. Then, the largest power of 2 that is lower than the message is:
- 2: if lower or equal than 2
- 1: if strictly lower than 2
So, if this changes, then also the expected squares will change.  Correct me please if I am wrong.

Reply:
left a comment after that linked comment. I interpret the rules to be `<=` instead of `<`

Reply:
Resolving, thanks a lot for your time and help.